### DISCUSSION TITLE
Registry errors

### DISCUSSION THREAD

**Discussion by triccs** [mandrean/cw-optimizoor#30] [Q&A] [answered]
Anytime I run 'cargo cw-optimizoor' I'm getting failed builds with a bunch of registry errors, any help is appreciated.

![image](https://user-images.githubusercontent.com/105668529/211208230-95a13554-66c6-40bb-80df-77db3cdec508.png)

**Comment by triccs** [comment, marked as answer]
my wasm target wasn't up to date so fixed by readding it

rustup target add wasm32-unknown-unknown

**Reply by mandrean** [reply]
Hey @triccs, happy you solved it already!

It's also mentioned briefly in the installation instructions:

https://github.com/mandrean/cw-optimizoor#installation

**Reply by triccs** [reply]
I had been following those (worked perfectly thanks) but I must have deleted something while debugging

**Reply by mandrean** [reply]
I think it has happened me before when I upgraded (or switched?) Rust to a different version using `rustup`; then I too needed to manually re-add the WASM target
